chromium_src/chrome/common/url_constants.cc

```diff
@@ -229,7 +229,7 @@ const char kSyncLearnMoreURL[] =
     "https://community.brave.com";
 
 const char kUpgradeHelpCenterBaseURL[] =
-    "https://community.brave.com";
+    "https://community.brave.com/?p=update_error&error=";
```

Comment:
`kUpgradeHelpCenterBaseURL` is used by `UpdateCheckDriver::OnUpgradeError() `
```
  base::string16 html_error_msg =
      base::StringPrintf(L"%d: <a href='%ls0x%X' target=_blank>0x%X</a>",
                         error_code_, base::UTF8ToUTF16(
                             chrome::kUpgradeHelpCenterBaseURL).c_str(),
                         hresult_, hresult_);
```
W/o this url change, broken link address is created.

Reply:
shouldn't these point to something more useful? They all just go to the community home page

Reply:
I guess that's a separate issue

Reply:
@bridiver @Brave-Matt is working on building out support pages for these.